### PR TITLE
QUIZ-61 Setup question title on create quiz page

### PR DESCRIPTION
**Pull request by BenLanglois**
Also refactored the instructor multiple choice button to share the same input border as the new question title input.

https://user-images.githubusercontent.com/28707918/118052410-ac09a180-b350-11eb-8bd6-b04458a33baa.mp4

### CODE REVIEW

File: src/instructor/InputBorder.tsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { StretchRow } from 'src/components/Containers';
+import styled from 'styled-components';
+import { BorderProps, ColorProps, border, color } from 'styled-system';
+
+export const InputBorder = styled(StretchRow)<BorderProps | ColorProps>`
+  border-radius: 16px;
+  padding: 0.5em;
+  overflow: hidden;
+  border: 2px solid ${props => props.theme.colors.blue};
+  background-color: ${props => props.theme.colors.lightGray};
```
Comment (alexgirard) [inline src/instructor/InputBorder.tsx:10]: nit: I prefer to use `themeGet` here

**Review by alexgirard — APPROVED** [review]
💯

**Comment by alexgirard** [comment]
wait what's going on with the question type buttons?

File: src/instructor/Questions.tsx [hunk 2]
```diff
@@ -110,7 +119,14 @@ export const Questions = ({
                     label="Question:"
                     toggle={showQuestion && ShowQuestionToggle}
                   >
-                    <p>question input</p>
+                    <InputBorder>
+                      <PlaceholderInput
```
Comment (alexgirard) [inline src/instructor/Questions.tsx:123]: why is the left padding so large in the video?

**Comment by BenLanglois** [comment]
> wait what's going on with the question type buttons?

It looks like I broke it in #20 when trying to fix an error message that I was seeing in the console. I created a PR to fix this: #23

File: src/instructor/Questions.tsx (hunk 2 above)
Comment (BenLanglois) [reply src/instructor/Questions.tsx:123]: It uses the same padding as the option labels component, but it looks like those labels are too large compared to the figma. I'll make both of them smaller.

**Review by alexgirard — APPROVED** [review]
👍

**Comment by heathermusson** [comment]
@BenLanglois This has been approved for a while, can you please merge.

**Pull request merged**